test/unit/mdc-menu/simple.foundation.test.js

```diff
@@ -20,23 +20,55 @@ import td from 'testdouble';
 import {captureHandlers, verifyDefaultAdapter} from '../helpers/foundation';
 import {setupFoundationTest} from '../helpers/setup';
 import {createMockRaf} from '../helpers/raf';
-import MDCSimpleMenuFoundation from '../../../packages/mdc-menu/simple/foundation';
-import {cssClasses, strings, numbers} from '../../../packages/mdc-menu/simple/constants';
+import {MDCSimpleMenuFoundation} from '../../../packages/mdc-menu/simple/foundation';
+import {cssClasses, strings, numbers, Corner} from '../../../packages/mdc-menu/simple/constants';
 
 function setupTest() {
   const {foundation, mockAdapter} = setupFoundationTest(MDCSimpleMenuFoundation);
   const size = {width: 500, height: 200};
-  const itemYParams = {top: 100, height: 20};
-  td.when(mockAdapter.hasClass('mdc-simple-menu')).thenReturn(true);
-  td.when(mockAdapter.hasClass('mdc-simple-menu--open')).thenReturn(false);
+  td.when(mockAdapter.hasClass(cssClasses.ROOT)).thenReturn(true);
+  td.when(mockAdapter.hasClass(cssClasses.OPEN)).thenReturn(false);
   td.when(mockAdapter.hasNecessaryDom()).thenReturn(true);
   td.when(mockAdapter.getNumberOfItems()).thenReturn(1);
   td.when(mockAdapter.getInnerDimensions()).thenReturn(size);
-  td.when(mockAdapter.getYParamsForItemAtIndex(0)).thenReturn(itemYParams);
 
   return {foundation, mockAdapter};
 }
 
+// Various anchor dimensions.
+const smallTopLeft = {height: 20, width: 40, top: 20, bottom: 40, left: 20, right: 60};
+const smallBottomLeft = {height: 20, width: 40, top: 920, bottom: 940, left: 20, right: 60};
+const smallBottomRight = {height: 20, width: 40, top: 920, bottom: 940, left: 920, right: 960};
+const smallCenter = {height: 20, width: 40, top: 490, bottom: 510, left: 480, right: 520};
+const smallAboveMiddleLeft = {height: 20, width: 40, top: 400, bottom: 420, left: 20, right: 60};
+const smallBelowMiddleLeft = {height: 20, width: 40, top: 600, bottom: 620, left: 20, right: 60};
+const wideCenter = {height: 20, width: 100, top: 490, bottom: 510, left: 450, right: 550};
```

Comment:
IIUC these configure the anchor to be the same width as the menu. Do we need some tests for when the anchor is wider than the menu to test which side the menu adheres to in LTR vs. RTL for cases where it overlaps the anchor?